contrib/k8s/dex-worker.yaml

```diff
@@ -0,0 +1,113 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: dex
+type: Opaque
+data:
+  key-secrets: ZUhoNGVIaDRlSGg0ZUhoNGVIaDRlSGg0ZUhoNGVIaDRlSGg0ZUhoNGVIZz0= # 32 x's base64 encoded twice.
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    app: dex
+    role: worker
+  name: dex-worker
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: dex
+        role: worker
+    spec:
+      containers:
+          - image: quay.io/coreos/dex
+            name: dex-worker
+            env:
```

Comment:
I think maybe you could replace the `--key-secrets` flag below with an env `DEX_WORKER_KEY_SECRETS`, and then use the downward API to pull the env var from a secret file:

```
- name: DEX_WORKER_KEY_SECRETS
  valueFrom:
    secretKeyRef:
      name: dex
      key: key-secrets
```